crates/turbopack-tests/tests/snapshot/typescript/tsconfig-extends-module-full-path/input/tsconfig.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "tsconfig-mod/tsconfig.json"
```

I think the test case could be improve by also testing omitting the `.json` extension and naming the file different than tsconfig

```suggestion
  "extends": "tsconfig-mod/config"
```